<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPasswordWidget</class>
 <widget class="QWidget" name="ResetPasswordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Verify Email To Reset Passowrd</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/qrc_images/assets/images/login_bg.png</normaloff>:/qrc_images/assets/images/login_bg.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ===== Background Gradient ===== */
QWidget#ResetPasswordWidget {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 #667EEA,
        stop: 1 #764BA2
    );
    font-family: &quot;Segoe UI&quot;;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QFrame" name="resetPasswordPanel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>385</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>650</width>
       <height>700</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#resetPasswordPanel {
    background-color: #ffffff;
    border-radius: 20px;
    padding: 30px;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0">
       <spacer name="verticalSpacer_Button_Top">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer_Buttons_Bottom">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>8</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_input">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="labelEmail">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #333333;
    font-size: 13pt;
    margin-top: 10px;
    margin-bottom: 4px;
    background: transparent;
}</string>
          </property>
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditRegisteredEmail">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>65</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid #E0E0E0;
    border-radius: 10px;
    padding: 8px 10px;
    background: rgba(250, 250, 250, 200); /* slightly transparent soft white */
    font-size: 11pt;
    color: #111111;                         /* user text color */
    font-style: normal;                      /* ensures user text is not italic */
}

QLineEdit:focus {
    border: 2px solid #667EEA;
    background: rgba(250, 250, 250, 220);   /* slightly brighter on focus */
}

QLineEdit::placeholder {
    color: rgba(50, 50, 50, 200);           /* placeholder text color */
    font-style: italic;                      /* placeholder remains italic */
}
</string>
          </property>
          <property name="placeholderText">
           <string>Enter registered email</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QPushButton" name="buttonSendCode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>55</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#buttonSendCode {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #6A89E5, stop:1 #8A5CCF);
    color: white;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-size: 11pt;
    font-weight: bold;
}

QPushButton#buttonSendCode:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #5872C7, stop:1 #7547B8);
    padding-top: 8px;
    padding-bottom: 12px;
}

QPushButton#buttonSendCode:pressed {
    background: #4A5AA3;
    padding-top: 12px;
    padding-bottom: 8px;
}

QPushButton#buttonSendCode:disabled {
    background: #AAAAAA;
    color: #666666;
    padding: 10px;
}</string>
          </property>
          <property name="text">
           <string>Send Verification code</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTimer">
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Timer before re-sending code again&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#labelTimer {
    background: transparent;
    color: #555;           /* soft gray, readable but not dominant */
    padding: 2px 4px;      /* small, compact */
    font-size: 9pt;        /* smaller text */
    font-weight: semi-bold;
    qproperty-alignment: AlignCenter;
}
QLabel#labelTimer[expired=&quot;true&quot;] {
    color: #2ECC71;   /* subtle green to indicate ‚Äúready‚Äù */
    font-weight: bold;
}
</string>
          </property>
          <property name="text">
           <string>60</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelCode">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #333333;
    font-size: 13pt;
    margin-top: 10px;
    margin-bottom: 4px;
    background: transparent;
}</string>
          </property>
          <property name="text">
           <string>Code</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditCode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>65</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid #E0E0E0;
    border-radius: 10px;
    padding: 8px 10px;
    background: rgba(250, 250, 250, 200); /* slightly transparent soft white */
    font-size: 11pt;
    color: #111111;                         /* user text color */
    font-style: normal;                      /* ensures user text is not italic */
}

QLineEdit:focus {
    border: 2px solid #667EEA;
    background: rgba(250, 250, 250, 220);   /* slightly brighter on focus */
}

QLineEdit::placeholder {
    color: rgba(50, 50, 50, 200);           /* placeholder text color */
    font-style: italic;                      /* placeholder remains italic */
}
</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter 4-digits code sent to your email</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QPushButton" name="buttonVerifyCode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>43</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>55</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#buttonVerifyCode {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #6A89E5, stop:1 #8A5CCF);
    color: white;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-size: 11pt;
    font-weight: bold;
}

QPushButton#buttonVerifyCode:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #5872C7, stop:1 #7547B8);
    padding-top: 8px;
    padding-bottom: 12px;
}

QPushButton#buttonVerifyCode:pressed {
    background: #4A5AA3;
    padding-top: 12px;
    padding-bottom: 8px;
}
QPushButton#buttonVerifyCode:disabled {
    background: #AAAAAA;
    color: #666666;
    padding: 10px;
}
</string>
          </property>
          <property name="text">
           <string>Verify</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_Buttons">
        <property name="spacing">
         <number>10</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QPushButton" name="buttonBackToSignIn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>55</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#buttonBackToSignIn {
    background: white;
    color: #667EEA;
    border: 2px solid #667EEA;
    border-radius: 10px;
    padding: 10px;
    font-size: 12pt;
    font-weight: bold;
}

QPushButton#buttonBackToSignIn:hover {
    background: #f0f2ff;       /* light purple/blue tint */
    padding-top: 8px;
    padding-bottom: 12px;       /* 3D lift effect */
}

QPushButton#buttonBackToSignIn:pressed {
    background: #e0e4ff;       /* darker when pressed */
    padding-top: 12px;
    padding-bottom: 8px;       /* pressed down effect */
}
</string>
          </property>
          <property name="text">
           <string>Back to Sign in</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_Title">
        <property name="spacing">
         <number>6</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="logoIcon">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#logoIcon {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #667EEA, stop:1 #764BA2);
    border-radius: 40px;
    min-width: 80px;
    min-height: 80px;
    max-width: 80px;
    max-height: 80px;
    font-size: 40px;
    color: white;
    qproperty-alignment: 'AlignCenter';
}
</string>
          </property>
          <property name="text">
           <string>üîÑÔ∏è</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="lableTitle">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #333333;
    font-size: 18pt;
    font-weight: bold;
    text-align: center;
    background: transparent;
}</string>
          </property>
          <property name="text">
           <string>Password Reset</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSubtitle">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#labelSubtitle{
    color: #666666;              /* gray color like HTML example */
    font-size: 12pt;              /* adjust size to fit QFrame */
    font-weight: normal;
    qproperty-alignment: 'AlignCenter';   /* center text horizontally */
    margin-bottom: 14px;
    background: transparent;        
}
</string>
          </property>
          <property name="text">
           <string>Remember your password next time</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="login_resources.qrc"/>
 </resources>
 <connections/>
</ui>
